README.md

```diff
@@ -36,26 +36,26 @@ For user-customizable oscillators online constructor is available at https://duk
 * Any types and combinations of logue-series can be injected in Anthologue.
-* Maximum number of Anthologue programs depends on their types and combinations and can vary from 25 to 76.
-* Due to logue-sdk parameter initialization specific, FM64 and Anthologue oscillators could alter program parameters on selection. Change the program after oscillator selection to make sure all parameters are loaded from the program to their default values.
-* With Anthologue only NTS-1 can utilize system BPM with play mode 3. All other -logue synths works the same way for both sequence modes: internal oscillator BPM initialized from the program and can be changed with assignable controllers only.
-* All 6 VCO of Anthologue are identical and sequentially chained with sync/ring mod/cross mod.
-* VCO 4-6 of Anthologue considered as a sub timbre, to utilize them either select a prologue program with sub timbre or force sub timbre and set with Sub On AC, Main/Sub Balance AC and Sub parameter.
-* Split sub timbre type is avalable for all models since it unilize 3 VCO at a time.
-* 6 VCO works stable only on NTS-1 with up to 2 FX.
-* Sub timbre is reset on program change. For prologue program - according to program settings. For other logues programs - Sub On: switched off, Main/Sub balance: center, VCO 4-6 are reset.
-* On -logues prologue program with timbre mode other than Split are loaded with sub timbre forcefully disabled on to avoid oscillator hang.
+* Maximum number of Anthologue programs depends on their types and combinations, and can vary from 25 to 76.
+* Due to logue-sdk parameter initialization implementation, FM64 and Anthologue oscillators may alter program parameters on selection. Change the program after oscillator selection to make sure all parameters are loaded from the program to their default values.
+* With Anthologue, only the NTS-1 can utilize system BPM with play mode 3. All other -logue synths work the same way for both sequence modes: internal oscillator BPM initialized from the program, and can only be changed with assignable controllers.
+* All 6 VCOs of Anthologue are identical, and sequentially chained with sync/ring mod/cross mod.
+* VCOs 4-6 of Anthologue are considered as a sub timbre. To utilize them, either select a Prologue program with sub timbre, or force sub timbre and set with Sub On AC, Main/Sub Balance AC and Sub parameter.
+* Split sub timbre type is available for all models since it utilizes 3 VCOs at a time.
+* 6 VCO is only stable on the NTS-1 with up to 2 FX.
+* Sub timbre is reset on program change. For Prologue program - according to program settings. For other -logues programs - Sub On: switched off, Main/Sub balance: center, VCO 4-6 are reset.
+* On -logues, Prologue programs with timbre mode other than Split are loaded with sub timbre forcefully disabled, in order to avoid oscillator hang.
```

Comment:
I hope I got the meaning here correct?

Reply:
@tervez Yep, all is correct. Thanks again! :)